src/helpers/Grid.ts

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export function useWindowSize() {
+  const [size, setSize] = React.useState([0, 0]);
+  React.useLayoutEffect(() => {
```

Comment:
I think we should use `useEffect` instead.
Ref: https://reactjs.org/docs/hooks-reference.html#uselayouteffect